lib/CollectionsTable/styles.scss

```diff
@@ -48,31 +49,64 @@ $collections-table-indent-level-support: 10;
 .collections-table__cell {
   border-top: $collections-table-border;
   border-bottom: $collections-table-border;
-
+  padding: ($collections-table-content-padding + 1) $collections-table-content-padding;
+  .collections-table__row--selected & {
+    border-top: $collections-table-selected-border;
+    border-bottom: $collections-table-selected-border;
+    padding: $collections-table-content-padding;
+  }
   &:first-child {
     padding: 0;
     border: 0;
   }
 
   &:first-child > .collections-table__cell-content {
-    padding: $collections-table-spacing;
+    padding: ($collections-table-spacing + 1) $collections-table-spacing ($collections-table-spacing + 1) ($collections-table-spacing + 1);
     border-radius: $collections-table-border-radius 0 0 $collections-table-border-radius;
     border-left: $collections-table-border;
     border-top: $collections-table-border;
     border-bottom: $collections-table-border;
+    .collections-table__row--selected & {
+      border-top: $collections-table-selected-border;
+      border-bottom: $collections-table-selected-border;
+      border-left: $collections-table-selected-border;
+      padding: $collections-table-spacing;
+    }
   }
 
   &:last-child {
     border-radius: 0 $collections-table-border-radius $collections-table-border-radius 0;
     border-right: $collections-table-border;
+    .collections-table__row--selected & {
+      border-top: $collections-table-selected-border;
+      border-bottom: $collections-table-selected-border;
+      border-right: $collections-table-selected-border;
+    }
   }
+
 }
 
 .collections-table__cell-content {
   display: flex;
   align-items: center;
   position: relative;
   min-width: 0;
+  .file-card {
```

Comment:
When normally deal with component composition styling we apply a class like `collections-table__file-card`. This can be seen in this stylesheet e.g. `collections-table__status-indicator`.

I'm down for this way to styling components within components but I just want to know if we're doing this moving forward so I can follow that pattern 😄 

------

I guess it's easier to manage if we just use the already defined classes...thoughts?

Reply:
@kyleharper Ah see I think there's an argument for both. If its a component we include ourselves then adding that composition styling makes total sense. 

However if its just a component that can live inside there, but isn't really part of it by default, and its totally up to the user then this way makes more sense. What do you think?

Reply:
By the latter scenario making more sense I mean that our components should just work with each other (within reason) without having to add extra classNames. 

Butttttttt I'm easy either way, just musings.

Reply:
Exactly my thoughts 👍 😄